app/controllers/users_controller.rb

```diff
@@ -7,7 +7,7 @@ def show
     @user = current_user
     # birthday = Happybirthday.born_on(current_user.birth_date)
     # @birthday = birthday.age.years_old
-    @user_information = UserInformation.find(params[:id])
+    @user_information = UserInformation.find_or_initialize_by(params[:id]) 
```

Comment:
末尾の無駄なスペースは削除するようにしましょう
```suggestion
    @user_information = UserInformation.find_or_initialize_by(params[:id])
```